playbooks/group_vars/grafana_servers/20-ldap_config.yaml

```diff
@@ -0,0 +1,34 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+
+grafana_ldap__servers__host: "{{ (glauth__service_ldap.listen.split(':')[0]
+                                  if glauth__service_ldap.listen is defined else None) }}"
+grafana_ldap__servers__port: "{{ (glauth__service_ldap.listen.split(':')[1]
+                                  if glauth__service_ldap.listen is defined else None) }}"
+grafana_ldap__servers__use_ssl: "{{ (glauth__service_ldap.tls | lower
+                                    if glauth__service_ldap.tls is defined else None) }}"
+grafana_ldap__servers__start_tls: "{{ (glauth__service_ldap.tls | lower
+                                      if glauth__service_ldap.tls is defined else None) }}"
```

Comment:
This has a few problems: the conditionals are incorrect, as they asume any defined value to be true (it should be `if (glauth__service_ldap.foo | d()) else None`), `use_ssl` should only be true if the `ldaps` service is activated, and more importantly: the `ldaps` service needs to be checked too. So it is quite complicated to do...

```suggestion
# Use LDAP if GLAuth is enabled.
_grafana_use_ldap: "{{ glauth__enable | d() | bool }}"
# Avoid errors.
_grafana_ldap_svc: "{{ (glauth__service_ldap | d()) or {} }}"
_grafana_ldaps_svc: "{{ (glauth__service_ldaps | d()) or {} }}"
# Use LDAPs if available.
_grafana_ldap_use_ldaps: "{{ _grafana_ldaps_svc.get('enabled') }}"
# Find bind address.
_grafana_ldap_listen: |-
  {{
    (
      _grafana_ldaps_svc if _grafana_ldap_use_ldaps else
      _grafana_ldap_svc
    ).get('listen') |
    mandatory(msg='Missing listen parameter in GLAuth configuration')
  }}

grafana_ldap__servers__host: "{{ _grafana_ldap_listen.split(':')[0] }}"
grafana_ldap__servers__port: "{{ _grafana_ldap_listen.split(':')[1] }}"

grafana_ldap__servers__use_ssl: "{{ _grafana_ldap_use_ldaps }}"
grafana_ldap__servers__start_tls: |-
  {{ not _grafana_ldap_use_ldaps or _grafana_ldap_svc.get('tls') }}

```